packages/codegen/generate.hooks.ts

```diff
@@ -252,34 +252,42 @@ export function generateReactHooks<
     }
 
     function useAllObjects (variables: QueryVariables, options?: QueryHookOptions<IUseObjectsQueryReturnType<GQLObject>, QueryVariables>) {
-        const { objs, count, error, loading, refetch, fetchMore, stopPolling } = useObjects(variables, options)
+        const { objs, count, error, loading, refetch: _refetch, fetchMore, stopPolling } = useObjects(variables, options)
         const [data, setData] = useState(objs)
-        const [allDataLoaded, setAllDataLoaded] = useState<boolean>(false)
+
+        const refetch: IRefetchType<GQLObject, QueryVariables> = useCallback((...args) => {
+            setData([])
+            return _refetch(...args)
+        }, [_refetch])
 
         useDeepCompareEffect(() => {
             setData([])
         }, [variables])
 
-        useDeepCompareEffect(() => {
-            if (data.length === count) {
-                setAllDataLoaded(true)
+        useEffect(() => {
```

Comment:
what happens if the response will be with error?

Reply:
added catch block and error state